server.js

```diff
@@ -78,6 +78,8 @@ if (app.get("env") === "dev") {
   cookie.secureProxy = false;
 }
 
+app.locals.SERVER_URL = AppConstants.SERVER_URL;
```

Comment:
I wonder if we should just do `app.locals.AppConstants = AppConstants` because we are likely to need many of the app constants values in the templates?

Reply:
Maybe if we want to keep things tight, something like this?
```js
app.locals.AppConstants = Object.freeze(Object.assign({SERVER_URL, STATIC_URL}, AppConstants))
```

Reply:
But don't worry about this for now.

Reply:
yeah, this sounds almost risky... Not sure I want Express to have access to _all_ those Top Secret(tm) env variables. I'd hate for some weird error route to potentially dump some template `app.locals` variables which include our API creds or something else.